### DISCUSSION TITLE
Question: Add Business account

### DISCUSSION THREAD

**Discussion by JFCCoding12** [mmohades/Venmo#65] [Q&A]
Just looking to see if it would be possible to use this API for venmo business accounts. You log in with normal account credentials so don't know how you can make requests come from the business account. Thoughts?

**Comment by mmohades** [comment]
It should be possible for doing the same things as a personal account. Have you tried it?

**Reply by JFCCoding12** [reply]
Just don’t know how to get the id of that account since it’s almost like a sub account using the same username and password as a personal account

**Reply by mmohades** [reply]
Does `client.my_profile().to_json()` help?

**Reply by JFCCoding12** [reply]
just lists my personal profile information doesn't mention anything about the business account

**Reply by JFCCoding12** [reply]
Was poking around with Charles Proxy on my phone and it looks like it makes the normal .../v1/account request but some how it responds with the business info when I click the business and makes the same request for my personal account when I click on that so I am not sure how it knows which account to show

**Reply by JFCCoding12** [reply]
Think I got it, /v1/users/identities seems to list the connected accounts

**Reply by JFCCoding12** [reply]
If you view my fork you can see the function I made that adds that functionality https://github.com/JFCCoding12/Venmo

**Reply by mmohades** [reply]
Nice! Can you include the response format of `/v1/users/identities` here?
Feel free to open a PR to https://github.com/mmohades/VenmoApiDocumentation and this repository.

**Reply by JFCCoding12** [reply]
Yep, I will go ahead and do that at some point tonight.
